### ISSUE TITLE
[my-result] description

### ISSUE THREAD

**Issue by tanfy929** [CVIU-CSU/M2MRF-Lesion-Segmentation#4]
Dear Author:
  I have tried to reproduce your work, and the parameter settings are consistent.However,the performance of the metrics on the test set is poor.I wonder if the code you uploaded to Github has been modified in the future?Or could you please tell me any details I might have overlooked?
  In addition,I ran the original code from the website and found that the HRNet_M2MRF_C seemed to perform worse than HRNet,and the results were quite different from those reported in the paper.The results of the two types of networks are shown in figures.Could you please tell me what the problem is?Am I missing something?
  Looking forward to your reply.Thank you!
![HRNet](https://user-images.githubusercontent.com/109494250/186561277-c02604ba-dd2e-48a3-9a55-797b4bfcbb73.jpg)
![HRNet_M2MRF_C](https://user-images.githubusercontent.com/109494250/186561285-0ecf3cc6-840e-4e1c-9396-b244e2ea129e.jpg)

**Comment by haotianll** [comment]
Hi, are these results from IDRiD testing set?
Do you use the pretrained model to get the results? Or do you train from the begining following the provided config file?

**Comment by tanfy929** [comment]
Yes,the results are from the IDRID dataset and I have used the pretrained model,with no modifications to the configuration files in the source code.
Perhaps you could send the log file of the code running?I can check the details.Thank you!

**Comment by haotianll** [comment]
How about the testing performance of model [here](https://github.com/CVIU-CSU/M2MRF-Lesion-Segmentation#idrid)?
Do you run the offline augmentation in "tools/augment.py"?

**Comment by tanfy929** [comment]
I ran the code on GitHub directly according to the 'Read Me' document, including the augmentation part, and the results are attached to the figures in the issue I raised.The results in the figures are the metrics from the IDRID test.

**Comment by haotianll** [comment]
Could you provide the testing performance of this given [model](https://drive.google.com/file/d/11YoorrgNds960WTNypDs4qissjgoLZd1/view)?

**Comment by tanfy929** [comment]
OK!
![M2MRF_C](https://user-images.githubusercontent.com/109494250/186661155-7701b275-850a-4b40-bf86-ac5db9c319f0.jpg)
![model](https://user-images.githubusercontent.com/109494250/186661165-6730a7d2-d98f-449a-b4bb-713b4e7a5827.jpg)

**Comment by haotianll** [comment]
This result looks correct.
Could you please provide the full training log of your previous result?

**Comment by tanfy929** [comment]
Ok! I have uploaded both log files of HRNet_M2MRF_C and HRNet.Thank you very much!

[HRNet_M2MRF_C-log.log](https://github.com/CVIU-CSU/M2MRF-Lesion-Segmentation/files/9424661/HRNet_M2MRF_C-log.log)
[HRNet-log.log](https://github.com/CVIU-CSU/M2MRF-Lesion-Segmentation/files/9424663/HRNet-log.log)

**Comment by haotianll** [comment]
Thank you!
How many gpus do you use for the training? The reported results are trained on 4 gpus with a total batch size of 4.

**Comment by tanfy929** [comment]
I used single GPU and the batch size is 1.Does batch will have such a big impact on the metrics?
I am not sure if there is something wrong with the parameter settings that I am not aware of.Would it be convenient for you to send me your running log?Or is it convenient for you to provide your email address?We can communicate further.Thank you!

**Comment by haotianll** [comment]
The batch size will affect layers like batch normalization. And I think your other settings are okay.
If batch size is 1, one alternative solution is to train the model with a smaller learning rate and more iterations, but I'm not sure whether it can achieve similar results as reported. 
For more questions, please contact me by haotianliu001@hotmail.com

**Comment by tanfy929** [comment]
All right! I'll try again
Thank you very much！

**Comment by haotianll** [comment]
You are welcome~
